test/evergreen.yml

```diff
@@ -2798,8 +2798,8 @@ buildvariants:
   expansions:
     make_command: PATH=/opt/mongodbtoolchain/v3/bin:$PATH make
     posix_configure_flags:
-      --enable-silent-rules --enable-python --enable-zlib --enable-snappy
-      --enable-strict --enable-static --prefix=$(pwd)/LOCAL_INSTALL
+      --enable-silent-rules --enable-python --enable-zlib
+      --enable-strict --enable-static --with-builtins=snappy
```

Comment:
Does that work? I thought that builds it statically.

Reply:
It does work but I'll try with only --enable-snappy.